### PR TITLE
[StyleCop] Fix warnings in ISlackAdapterOptions and SimpleSlackAdapterOptions files

### PR DESCRIPTION
**Pull request by ceciliaavila**
## Proposed Changes
- Add value documentation for properties.
- Change parameter name to lowerCase.

### CODE REVIEW

File: libraries/Microsoft.BotKit.Adapters.Slack/ISlackAdapterOptions.cs [hunk 1]
```diff
@@ -13,50 +13,57 @@ public interface ISlackAdapterOptions
...
         /// Gets or Sets an array of scope names that are being requested during the oauth process. Must match the scopes defined at api.slack.com.
         /// </summary>
+        /// <value>The Scopes array.</value>
         string[] Scopes { get; set; }
 
         /// <summary>
-        /// Gets or Sets the URL users will be redirected to after an oauth flow. In most cases, should be `https://<mydomain.com>/install/auth`.
+        /// Gets or Sets the URL users will be redirected to after an oauth flow. In most cases, should be `https://mydomain.com/install/auth`.
         /// </summary>
+        /// <value>The Redirect URL.</value>
```
Comment (matiasroldan6) [inline libraries/Microsoft.BotKit.Adapters.Slack/ISlackAdapterOptions.cs]: Sorry but it would be URI, not URL

File: libraries/BotKit Samples/Microsoft.Bot.Sample.Slack/SimpleSlackAdapterOptions.cs [hunk 2]
```diff
@@ -29,12 +29,12 @@ public SimpleSlackAdapterOptions(string verificationToken, string botToken)
 
         public string RedirectUri { get; set; }
```
Comment (GasparAcevedoZainSouthworks) [inline libraries/BotKit Samples/Microsoft.Bot.Sample.Slack/SimpleSlackAdapterOptions.cs]: Shouldn't this property have a _summary/comment_? Or these kind of changes go in another PR?

File: libraries/Microsoft.BotKit.Adapters.Slack/ISlackAdapterOptions.cs (hunk 1 above)
Comment (ceciliaavila) [reply libraries/Microsoft.BotKit.Adapters.Slack/ISlackAdapterOptions.cs]: Feedback applied. Thanks!

File: libraries/BotKit Samples/Microsoft.Bot.Sample.Slack/SimpleSlackAdapterOptions.cs (hunk 2 above)
Comment (ceciliaavila) [reply libraries/BotKit Samples/Microsoft.Bot.Sample.Slack/SimpleSlackAdapterOptions.cs]: I think StyleCop doesn't require these because they are already documented in the interface which they belong.
